.stackgenrc.ts

```diff
@@ -135,7 +137,7 @@ Bindings
 
     new ManifestEntry(project, 'CompileScript', {
       scripts: {
-        compile: 'tsup ./index.ts',
+        compile: `tsup --out-dir ${buildPath} ./index.ts`,
```

Comment:
tsup picks this up from tsconfig right?

Reply:
The `--out-dir` value? 

Not sure. It's not documented in the docs, only as `--help` flag:

```
  -d, --out-dir <dir>          Output directory (default: dist)
```

So by sound of that, I think it's not using tsconfig values.